base.css

```diff
@@ -1,10 +1,89 @@
-@import "intro.css";
-@import "index.css";
-
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
   font-size: 100%;
   font-family: "Roboto", sans-serif;
 }
+
+.gridLayout {
+  position: absolute;
+  min-width: 414px;
+  height: 100vh;
+  background-color: RGB(25, 30, 40);
+  display: flex;
+  flex-direction: column;
```

Comment:
I would've expected a grid, judging by the class name :D

Reply:
You are right! ;)  Just wanted to test you! ;D

THX

.gridLayout   (changed to flexLayout)